js/packages/mobile-wallet-adapter-protocol/src/createSIWSMessage.ts

```diff
@@ -0,0 +1,22 @@
+import { Buffer } from 'buffer'
+import { SignInPayload } from './types';
+import { SolanaSignInInputWithRequiredFields, createSignInMessageText } from '@solana/wallet-standard-util';
+
+export function createDefaultSolanaSignInParams(): Pick<SignInPayload, 'version' | 'chainId' | 'nonce' | 'issuedAt' > {
+    return {
+        version: '1',
+        chainId: '1',
+        nonce: Buffer.from(
+            crypto.getRandomValues(new Uint8Array(8))
```

Comment:
Sanity check here: Where does the `crypto` module come from? Does this work in react native app's that don't import some `crypto` polfyill?

Reply:
we chatted about this on slack but posting her for viz:

this does require that the react native app uses the `react native get random values` polyfill. this was already a requirement so nothing has changed there.